packages/contracts-bedrock/src/universal/FeeVault.sol

```diff
@@ -76,7 +76,10 @@ abstract contract FeeVault {
         emit Withdrawal(value, RECIPIENT, msg.sender, WITHDRAWAL_NETWORK);
 
         if (WITHDRAWAL_NETWORK == WithdrawalNetwork.L2) {
-            SafeCall.send(RECIPIENT, gasleft(), value);
+            require(
+                SafeCall.send(RECIPIENT, gasleft(), value),
```

Comment:
I kept this to minimize the diff, but I think there's a good argument for just checking the status of `(bool success, ) = RECIPIENT.call(hex"")`. 

The SafeCall lib arguably led to this bug being introduced by hiding the silent failure.

Reply:
And that we definitely won't be setting the `RECIPIENT` address to a contract that returns so much memory that we OOG here.

Reply:
This is true, i suppose there was no invariant defined here

Reply:
But you are correct that the bug here results in the total processed variable being incorrect which is not ideal

Reply:
depending on what the invariant is, you could also do:

```
bool success = safecall();
if (success) { totalProcessed + balance }
```

Reply:
Do you mean `SafeCall.send()` where you use `safecall()?

Reply:
yes, was just typing out pseudocode

Reply:
So I ended up doing the following, because: 
1. we can remove an import
2. reverting on failure is consistent with how solidity handles a failure if the withdrawal is to L1 via `L2StandardBridge.bridgeETHTo()`. 

```
  (bool success, ) = RECIPIENT.call{ value: value }(hex"");
  require(success, "FeeVault: failed to send ETH to L2 fee recipient");
```

Reply:
I don't think removing an import is a particularly strong argument but not super opinionated, I do think using safecall is generally better because it will be cheaper

Reply:
I tested it and `SafeCall.send()` save 31 gas vs the current implementation. I would prefer to stick with this more explicit approach.